docs/pages/reference/session-recording.mdx

```diff
@@ -0,0 +1,162 @@
+that cannot decrypt the data flowing through it.
+
+Additionally, the credentials that the proxy uses to decrypt the SSH connection are
+provided via SSH Agent Forwarding, so Agent Forwarding must be enabled to record at
+the Proxy Server.
+
+## Synchronous recording
+
+When synchronous recording is enabled, the Teleport component doing the recording
+(which may be the node or the proxy depending on your configuration) submits each
+recording event to Teleport's Auth Server as it occurs. In this mode, failure to
+emit a recording event is considered fatal - the session will be terminated if an
+event cannot be recorded. This makes synchronous recording best suited for highly
+regulated environments where you need to be confident that all data is recorded.
+This also means that you need a reliable and low-latency connection to the Auth
+Server for the duration of the session to ensure that the session isn't interrupted
+or terminated due to temporary connection loss.
+
+In synchronous recording modes, the Auth Server receives a stream of recording
+events and is responsible for assembling them into the final artifact and uploading
+it to the storage backend. Since data is streamed directly to the Auth Server,
+Teleport administrators don't need to be concerned with disk space on their nodes
+and proxies, as no recording data is written to those disks.
+
+## Asynchronous recording
+
+When asynchronous, recording events are written to the local filesystem
+during the session. When the session completes, Teleport assembles the parts into a
+complete recording and submits the entire recording to the Auth Server for storage.
+
+Since recording data is flushed to disk, administrators should be careful to ensure
+that the system has enough disk space to accommodate the expected number of Teleport
+sessions. Additionally, since recording data is temporarily stored on disk, there
+is a greater chance that it can be tampered with, deleted, or otherwise corrupted
+before the upload completes.
+
+The advantage of asynchronous recording is that it doesn't require a persistent
+connection to the Auth Server. For example, an SSH session can continue to operate
+even if Teleport's Auth Server goes down. When the session completes Teleport will
+attempt to upload the recording to the Auth Server. If the Auth Server is still
+unavailable, Teleport has built-in retry and backoff mechanisms that will upload
+the artifact when the Auth Server comes back online. Additionally, asynchronous
+recording is well-suited towards recording sessions that are extra chatty or in
+environments where the connection to the auth server is unreliable or high-latency.
+
+## Storage
+
+Session recordings are stored in Teleport's audit sessions backend, which is
+specified by the `audit_sessions_uri` field in the `teleport.yaml` [configuration file](config.mdx).
+Teleport current supports the following session storage backends:
```

Comment:
Should we list the values you'd put in the configuration file below?

Reply:
Personally, I don't think so. There's quite a few permutations and it's already covered in detail in the config reference. This page was more meant to be about how it works rather than how to configure it.